projects/help/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@blackbaud/skyux-lib-help",
-  "version": "8.0.0",
+  "version": "9.0.0-alpha.0",
   "peerDependencies": {
-    "@angular/common": "^15.2.9",
-    "@angular/core": "^15.2.9",
+    "@angular/common": "^16.1.8",
+    "@angular/core": "^16.1.8",
     "@blackbaud/help-client": "^3.1.1",
-    "@skyux/config": "^8.0.0",
-    "@skyux/core": "^8.0.0"
+    "@skyux/config": "^9.0.0-alpha.0",
+    "@skyux/core": "^9.0.0-alpha.0"
   },
   "dependencies": {
-    "tslib": "^2.5.0"
+    "tslib": "^2.6.1"
```

Comment:
We didn't up this in the monorepo for SKY UX proper. Should we have? When I looked at the `tslib` changes - I didn't see anything that would stop `2.5.x` from working.